src/dom.js

```diff
@@ -827,3 +830,15 @@ export function isFullscreenElement(element) {
   }
   return false;
 }
+
+/**
+ * Returns true if node is not disabled.
+ *
+ * IE8 can return false positives, see {@link matches}.
+ * @param {!Element} element
+ * @return {boolean}
+ * @see https://www.w3.org/TR/html5/forms.html#concept-fe-disabled
+ */
+export function isEnabled(element) {
+  return !(element.disabled || matches(element, DISABLED_PSEUDOCLASS_));
```

Comment:
FWIW, I'm not a big fan of constants that redirect CSS names and identifiers. I'd much rather have `matches(element, ':disabled')` inlined.